script/test/Swap.sol

```diff
@@ -7,18 +7,31 @@ import { Script } from "script/utils/Script.sol";
 import { Chain } from "script/utils/Chain.sol";
 
 import { IBroker } from "mento-core/contracts/interfaces/IBroker.sol";
+import { IStableToken } from "mento-core/contracts/interfaces/IStableToken.sol";
 import { IExchangeProvider } from "mento-core/contracts/interfaces/IExchangeProvider.sol";
 import { IERC20Metadata } from "mento-core/contracts/common/interfaces/IERC20Metadata.sol";
 import { BiPoolManager } from "mento-core/contracts/BiPoolManager.sol";
 import { IBiPoolManager } from "mento-core/contracts/interfaces/IBiPoolManager.sol";
 
+interface IMockERC20 {
+  // need to define this because we don't have a single interface with all this methods
```

Comment:
Have you considered using the contract you've created? 
You can cast an address as a contract too, not just as an interface. The MockERC20 contract inherits from Ownable, which means it has the owner function, and it also inherits from ERC20, so it has all the other necessary functions.

Reply:
ah yeah didn't think of this  🤦🏽‍♂️ updated